Content.Client/UserInterface/Systems/Guidebook/GuidebookUIController.cs

```diff
@@ -208,4 +219,41 @@ private void RecursivelyAddChildren(GuideEntry guide, Dictionary<string, GuideEn
             RecursivelyAddChildren(child, guides);
         }
     }
+    private void PopOut()
+    {
+        if (_guideWindow == null)
+        {
+            return;
+        }
+
+        var monitor = _clyde.EnumerateMonitors().First();
+
+        ClydeWindow = _clyde.CreateWindow(new WindowCreateParameters
+        {
+            Maximized = false,
+            Title = "Guidebook",
+            Monitor = monitor,
+            Width = 750,
+            Height = 700
+        });
+        var control = _guideWindow.Guidebook;
+        control.Orphan();
+        _guideWindow.Dispose();
+        _guideWindow = null;
+
+        ClydeWindow.RequestClosed += OnRequestClosed;
+
+        ClydeWindow.DisposeOnClose = true;
+        var Root = _uiManager.CreateWindowRoot(ClydeWindow);
+        Root.AddChild(control);
+
+        control.PopOutButton.Disabled = true;
+        control.PopOutButton.Visible = false;
+    }
+
+    private void OnRequestClosed(WindowRequestClosedEventArgs obj)
```

Comment:
This needs to untoggle the Guidebook UI button as well or it will get desynced when you close the popped out window.